.github/workflows/issue-triage.yml

```
@@ -42,10 +42,12 @@ jobs:
             echo "issue_id=$ISSUE_ID" >> "$GITHUB_OUTPUT"
         env:
           GH_TOKEN: ${{ steps.app-token.outputs.token }}
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
       - name: Update issue with website area field
         run:
           gh project item-edit --project-id $PROJECT_ID --id $ISSUE_ID
           --field-id $AREA_FIELD_ID --single-select-option-id $AREA_WEBSITE
         env:
           GH_TOKEN: ${{ steps.app-token.outputs.token }}
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Should we just rip out the app token and exclusively use the PAT token instead?

```suggestion
          GH_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

I'm not sure the scopes that one has, but we can try it. We also have `ADD_TO_PROJECT_PAT` in the secrets that for sure has the projectsV2 scope but idk if it has the others necessary for the repo/issue queries.

---

updated to use the `ADD_TO_PROJECT_PAT`